engine/src/flutter/ci/builders/linux_web_engine_test.json

```diff
@@ -362,10 +367,14 @@
             "--suite=firefox-dart2js-canvaskit-engine",
             "--suite=firefox-dart2js-canvaskit-canvaskit",
             "--suite=firefox-dart2js-canvaskit-ui",
+            "--suite=firefox-dart2wasm-skwasm-ui",
```

Comment:
This suite has `enable-ci: false`, yet it's showing up in `--copy-artifacts`. The reason is this list is generated from `suites` not `filteredSuites`. Not sure if intentional or not.

Reply:
Good catch, thanks! I'll fix it.